### ISSUE TITLE
Harakavy's Yiddish-English dictionary

### ISSUE THREAD

**Issue by bansp** [freedict/fd-dictionaries#21]
https://github.com/dtolpin/harkavy

No licensing statement there, but I think the data is in the public domain.

**Comment by lwm** [comment]
@bansp, [IANAL](https://en.wikipedia.org/wiki/IANAL) but I think full copyright is asserted when the license is not explicitly stated. You'd need to raise an issue at that repository to find out exactly what it is? Not sure.

**Comment by bansp** [comment]
Yes, you're not a lawyer, but you can look up the publication date of the dictionary and form a reasonable assumption on (a.o) that basis :-)

**Comment by lwm** [comment]
> but you can look up the publication date of the dictionary

Well, I just searched a little and saw ([source](https://www.cs.uky.edu/~raphael/yiddish/harkavy/index.utf8.html)):

> Apparently, later editions of this dictionary are copyright around 1928. I will be happy to make arrangements with the copyright holders on application; please contact raphael @cs.uky.edu.

So I thought I would leave a comment.
